backend/services/ws.py

```
@@ -177,10 +186,16 @@ async def shutdown(self):
        logger.info("🛑 Shutting down WebSocket manager...")
 
      # cancel monitor
-       self.monitor_task.cancel()
+       if self.monitor_task:
+          self.monitor_task.cancel()
+          try:
+             await self.monitor_task
+          except asyncio.CancelledError:
+             pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(asyncio.CancelledError): ...`

(SIM105)